backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/interiminvoice/contract/C_Flatrate_Term_CreateInterimContract_StepDef.java

```diff
@@ -50,8 +50,8 @@ public C_Flatrate_Term_CreateInterimContract_StepDef(
 		this.flatrateTermTable = flatrateTermTable;
 	}
 
-	@And("create interim contract for modular contract")
-	public void create_interim_contract(@NonNull final DataTable dataTable)
+	@And("invoke \"C_BPartner_InterimContract_Create\" process:")
+	public void invokeC_BPartner_InterimContract_Create_process(@NonNull final DataTable dataTable)
```

Comment:
I think this is not correct for `S0306_200` , see `de.metas.contracts.modular.interceptor.C_Flatrate_Term.createInterimContractIfNeeded` vs `de.metas.contracts.modular.interim.invoice.process.C_BPartner_InterimContract_Create`